.devops/nix/package.nix

```diff
@@ -222,8 +228,8 @@ effectiveStdenv.mkDerivation (
     # TODO(SomeoneSerge): It's better to add proper install targets at the CMake level,
     # if they haven't been added yet.
     postInstall = ''
-      mv $out/bin/main $out/bin/llama
-      mv $out/bin/server $out/bin/llama-server
+      mv $out/bin/main${executableSuffix} $out/bin/llama
+      mv $out/bin/server${executableSuffix} $out/bin/llama-server
```

Comment:
This has been brought up before, it's really awkward we maintain a custom intsallPhase: the cmakelists need to be updated